src/main/main.c

```diff
@@ -248,7 +247,12 @@ void init(void)
 
     dmaInit();
 
-    serialInit(&masterConfig.serialConfig, feature(FEATURE_SOFTSERIAL));
+#ifdef AVOID_UART2_PWM_PPM
```

Comment:
Maybe it should be called  something like?
AVOID_UART2_FOR_PWM_PPM

I dont know...just thinking to prevent confusions in the future